src/test/java/org/jenkinsci/plugins/workflow/cps/CpsScmFlowDefinitionTest.java

```diff
@@ -195,6 +201,32 @@ public class CpsScmFlowDefinitionTest {
         r.assertLogContains("version one", b);
     }
 
+    @Test public void lightweight_brach_parametrised() throws Exception {
```

Comment:
```suggestion
    @Issue("JENKINS-42971")
    @Test public void lightweight_branch_parametrised() throws Exception {
```

(also, https://en.wiktionary.org/wiki/parametrise#Alternative_forms left me rather baffled)

Reply:
sorry for that, its fixed